Feature/pop up dish description

Rhilim opened the pull request.

Rhilim committed:
1

Rhilim committed:
1

Rhilim committed:
1

Rhilim committed:
modal

Rhilim committed:
close oprions added

Rhilim committed:
Update pop-up-recipe.css

biggujo commented:
Fix sizing:
– https://github.com/biggujo/project-cooking/assets/37770603/ac3a3ea1-19a4-442a-95b4-f49ebc3b74a7

Add markup:
– https://trello.com/c/Dv7qtblt/19-pop-up-card-dish-description

biggujo reviewed (changes requested):
Fix sizing:
– https://github.com/biggujo/project-cooking/assets/37770603/ac3a3ea1-19a4-442a-95b4-f49ebc3b74a7

Add markup:
– https://trello.com/c/Dv7qtblt/19-pop-up-card-dish-description

Rhilim committed:
modal markup done for mobile

Rhilim committed:
Merge branch 'main' into feature/pop-up-dish-description

Rhilim committed:
icons svg pulled

biggujo committed:
Fix SVG import path

biggujo commented:
![](https://github.trello.services/images/mini-trello-icon.png) [pop-up card dish description](https://trello.com/c/Dv7qtblt/19-pop-up-card-dish-description)

biggujo commented:
1. Fix height on small screen 
![image](https://github.com/biggujo/project-cooking/assets/37770603/233f0950-878d-44d5-808a-a93ce819c7e0)

2. Add adaptive width change for PC & tablet

Rhilim committed:
markup added

close svg not working dor some reason (please check)

also for Tablet and Desktop screen we have to change blocks order:
hastgat list and rateing string has to be placed in one row

Rhilim committed:
Merge branch 'feature/pop-up-dish-description' of https://github.com/biggujo/project-cooking into feature/pop-up-dish-description

Rhilim committed:
fix - tried to return lost changes

Rhilim committed:
fixed

Still have to fixed close-svg and for Tablet and Desktop manage to reorder blocks  - hasttag list and rating list

biggujo commented:
https://github.com/biggujo/project-cooking/assets/37770603/65224ab7-632b-48c0-8b16-e19ff14c8640

Fix sizing

Rhilim committed:
close btn fixed and modal width

Rhilim committed:
fixed media

Rhilim committed:
fixed Tablet and Desktop css element order

Rhilim committed:
fixed width

biggujo committed:
Refactoring: fix modal width, add color variables

biggujo committed:
Undone changes in icons.svg

biggujo committed:
Bring test function to test.js back
